### ISSUE TITLE
IntelliSense in function name shows variables that are local to other functions

### ISSUE THREAD

**Issue by bals88** [Microsoft/python-language-server#889]
**Os Version:** Windows 10 - 10.0.17763 Build 17763
**Extension Version:** Python 2019.3.6352
**Language Server Version:** Microsoft Python Language Server version 0.2.43.0
**Reproducible Code:**
```
def x():
    testVariable = "Test"

def test():
    test2 = 0
```
When I am naming the test function, it shows testVariable as an option in IntelliSense.
![Python Intellisense bug](https://user-images.githubusercontent.com/20808504/55615489-6bbeb900-57b1-11e9-8bc6-1c2a493b03a6.jpg)

**Comment by jakebailey** [comment]
I believe this is a duplicate of #843, which will be fixed in #819.

**Comment by bals88** [comment]
Sorry for creating a duplicate. I had checked but had not found that thread.
